fe/fe-core/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java

```diff
@@ -1438,9 +1758,20 @@ public void process(List<AlterClause> alterClauses, String clusterName, Database
                 // modify table properties
                 // do nothing, properties are already in propertyMap
             } else if (alterClause instanceof CreateIndexClause) {
-                processAddIndex((CreateIndexClause) alterClause, olapTable, newIndexes);
+                CreateIndexClause clause = (CreateIndexClause) alterClause;
+                processAddIndex(clause, olapTable, newIndexes);
+                if (clause.getIndexDef().getIndexType() == IndexDef.IndexType.BLOOMFILTER) {
+                    createBfJob(db.getId(), olapTable, indexSchemaMap, propertyMap, newIndexes, false);
+                    return;
+                }
             } else if (alterClause instanceof DropIndexClause) {
-                processDropIndex((DropIndexClause) alterClause, olapTable, newIndexes);
+                DropIndexClause clause = (DropIndexClause) alterClause;
+                if (getIndexTypeByName(clause, olapTable) == IndexDef.IndexType.BLOOMFILTER) {
+                    createBfJob(db.getId(), olapTable, indexSchemaMap, propertyMap, newIndexes, true);
```

Comment:
createBfJob is right?

Reply:
createBfJob is a new Function that responsible for bloomfilter index create/drop